meta-arago-distro/conf/layer.conf

```diff
@@ -14,8 +14,6 @@ LAYERDEPENDS_meta-arago-distro = " \
     openembedded-layer \
     networking-layer \
     meta-python \
-    linaro-toolchain \
-    meta-optee \
     qt5-layer \
     browser-layer \
```

Comment:
I think we should remove this as well @chombourger

Reply:
that would be fine by me

Reply:
You mean browser-layer ?

Reply:
yeah